chore(docs/examples): Updating Tempo X Node.js (#3960)
kubeshop/tracetest: docs/docs/examples-tutorials/recipes/running-tracetest-with-grafana-tempo-pokeshop.mdx

## Prerequisites

**Tracetest Account**:

- Sign up to [`app.tracetest.io`](https://app.tracetest.io) or follow the [get started](/getting-started/installation) docs.
- Have access to the environment's [agent API key](https://app.tracetest.io/retrieve-token).

**Docker**: Have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine.

## Run This Quckstart Example

The example below is provided as part of the Tracetest project. You can download and run the example by following these steps:

Clone the Tracetest project and go to the Grafana Tempo Node.js Quickstart:

```bash
git clone https://github.com/kubeshop/tracetest
cd tracetest/examples/tracetest-grafana-tempo-pokeshop
```

Follow these instructions to run the quick start:

1. Copy the `.env.template` file to `.env`.
2. Fill out the [TRACETEST_TOKEN and ENVIRONMENT_ID](https://app.tracetest.io/retrieve-token) details by editing your `.env` file.
3. Run `docker compose run tracetest-run`.
4. Follow the links in the output to view the test results.

> Follow along with the sections below for an in detail breakdown of what the example you just ran did and how it works.

Reviewer: ```suggestion
Follow the sections below for a detailed breakdown of what the example you just ran did and how it works.
```

## Project Structure

The project is built with Docker Compose.

### 1. Tracetest

The project contains [Tracetest Agent](/getting-started/installation#install-the-tracetest-agent), [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main), and a Node.js app.

The `docker-compose.yaml` file in the root directory of the quick start runs the Node.js app and the [Tracetest Agent](/concepts/agent) setup.

> Configure the `.env` like shown below.

Reviewer: ```suggestion
Configure the `.env` as shown below.
```
